### DISCUSSION TITLE
Add Sleep Timer

### DISCUSSION THREAD

**Discussion by shahariarnavil** [KRTirtho/spotube#257] [General]
u can add some extra feture that stop music after selected duration or after playing selected no of songs

**Comment by KRTirtho** [comment]
That's a really nice idea
Can you please create a [Feature Request](https://github.com/KRTirtho/spotube/issues/new?assignees=KRTirtho&labels=enhancement%2Cgood+first+issue&template=new_feature.yml&title=%5BFeature%5D+%3Ctitle%3E) out of it?

**Reply by saptarshi20022** [reply]
Three optional timers of 5mins, end of track and 10mins like the existing 15mins, would be really great to have!

**Comment by plsgivemeachane** [comment]
Really need timer

**Comment by plsgivemeachane** [comment]
But I think it should be to max 1 hours

**Comment by plsgivemeachane** [comment]
Oh it already there sorry🐧
